app/helpers/api/ordapi/postOrderItem.js

```diff
@@ -14,10 +14,10 @@ const formatPostData = ({
   formData,
 }) => ({
   ...selectedPrice,
```

Comment:
so serviceRecipient === serviceRecipientName if serviceRecipientName exists
but if it doesnt we set serviceRecipient to an object with a name propery

does that mean serviceRecipient will either be a string or an object (assuming serviceRecipientName is a string name and not an object)?

Reply:
`serviceRecipient` is always an object but we should only include it in the request to the API if we have the name and/or ID of the service recipient.

To be considered a valid request by the API, a service recipient must contain both a name and ID but a service recipient isn't always required. While we should never have only a name or an ID I thought it made sense to use an `or` condition rather than `and` so that the API validation is triggered if we ever encounter that situation, due to developer error say. Including a service recipient without a name or ID will always trigger the API validation but it will never be valid so shouldn't be included in this case.

Does that clarify things at all?